* Interactive Tree Of Life: https://itol.embl.de/
* NCBI Conserved domains: https://www.ncbi.nlm.nih.gov/Structure/cdd/wrpsb.cgi
* Multiple alignment using T-Coffee: http://tcoffee.crg.cat/
* Boxshade: http://www.ch.embnet.org/software/BOX_form.html
* RNAfold web server: http://rna.tbi.univie.ac.at//cgi-bin/RNAWebSuite/RNAfold.cgi
* RNA Secondary Structure Visualization: http://rna.tbi.univie.ac.at/forna/
* KEGG view: https://pathview.uncc.edu/
* L1000CDS: https://amp.pharm.mssm.edu/L1000CDS2/#/index
* CCLE: https://www.gtexportal.org/home/gene/NCOA4
* Cancer Dependency Map: https://depmap.org/portal/
* DepMap Portal: https://depmap.org/portal/depmap/
* NIH cancer treatment: https://www.cancer.gov/about-cancer/treatment
* Broad Institute Cancer Cell Line Encyclopedia (CCLE): https://portals.broadinstitute.org/ccle/page?gene=ZNF132
* Meta in R, support research with multi-evidences: https://bookdown.org/MathiasHarrer/Doing_Meta_Analysis_in_R/
* UCSC Toil RNAseq Recompute Compendium: https://xenabrowser.net/datapages/?host=https://toil.xenahubs.net
* Transcript level TCGA vs GTEx: https://xenabrowser.net/datapages/?host=https://toil.xenahubs.net
* TCGA-TPM-expression are produced at UCSC https://xenabrowser.net/datapages/?host=https://toil.xenahubs.net
* R heatmap: https://www.datanovia.com/en/lessons/heatmap-in-r-static-and-interactive-visualization/
* Online Venn: http://bioinformatics.psb.ugent.be/cgi-bin/liste/Venn/calculate_venn.htpl
* The SIGnaling Network Open Resource: https://signor.uniroma2.it/relation_result.php?id=Q13772
* Pathway common search: https://apps.pathwaycommons.org/search?type=Pathway&q=NCOA4
* A Summary of Missense Prediction Methods, http://acrmwww.biochem.ucl.ac.uk/saap/methods/
* SAAPdap - Single Amino Acid Polymorphism data analysis pipeline: http://acrmwww.biochem.ucl.ac.uk/saap/dap/
* Domain Graph, version 2.0: http://dog.biocuckoo.org/down.php
* Illustrator for Biological Sequences (IBS): http://ibs.biocuckoo.org/download.php
* MotifMap: genome-wide maps of regulatory elements: http://motifmap.ics.uci.edu/
* Jvarkit : Java utilities for Bioinformatics: http://lindenb.github.io/jvarkit/
* FGF6-blood: https://github.com/Shicheng-Guo/2LOF/tree/master/man
* ANNOVAR: https://github.com/Shicheng-Guo/HowtoBook/tree/master/ANNOVAR
* Metatranscriptomics analysis using microbiome RNA-seq data: https://galaxyproject.github.io/training-material/topics/metagenomics/tutorials/metatranscriptomics/tutorial.html

* EMBL-EBI: Protein Functional Analysis: https://www.ebi.ac.uk/Tools/pfa/
* ComplexHeatmap tutorial 1: https://jokergoo.github.io/ComplexHeatmap-reference/book/heatmap-annotations.html
* ComplexHeatmap tutorial 2: https://github.com/Shicheng-Guo/ComplexHeatmap
* miRNet: https://www.mirnet.ca/miRNet/upload/MirUploadView.xhtml
* Power estimation: http://csg.sph.umich.edu/abecasis/cats/gas_power_calculator/
* RNAfold web server ([example](https://github.com/Shicheng-Guo/RNAfold/blob/master/hsa-mir-323b/readme.md)): http://rna.tbi.univie.ac.at/cgi-bin/RNAWebSuite/RNAfold.cgi
* miRNA-SNP-seed region, mature: http://bioinfo.life.hust.edu.cn/miRNASNP2/index.php
* Locus room for manhattan plot: http://locuszoom.org/
* BIS180L A Lab Course at UC Davis to introduce Genetics and Genomics: https://jnmaloof.github.io/BIS180L_web/slides/

